src/TransformOperationExecutor.ts

```diff
@@ -425,6 +429,15 @@ export class TransformOperationExecutor {
         return this.options.groups.some(optionGroup => groups.indexOf(optionGroup) !== -1);
     }
 
+    private applyIncludePolicy(keys: string[], object: any, include: Include): string[] {
+        if (include === Include.DEFAULT) {
+            return keys;
+        } else if (include === Include.NON_NULL) {
+            return keys.filter((it: string) => object[it] !== null);
+        } else {
+            throw new Error(`Unsupported Include policy ${include}`);
+        }
```

Comment:
Suggested changes:

```typescript
if (include === Include.DEFAULT) {
    return keys;
}

if (include === Include.NON_NULL) {
    return keys.filter((it: string) => object[it] !== null);
}

throw new Error(`Unsupported Include policy ${include}`);
```

The variable names `include` and `it` are not clear enough. Please consider changing them later.